ci/deps/pyspark.yml

```diff
@@ -1 +1 @@
-pyspark>=2.4.3
```

Comment:
hmm do we not care about older pyspark? (ok by me) but we should make this clear

Reply:
We still have https://github.com/ibis-project/ibis/blob/master/ci/deps/pyspark-min.yml to test lower version of PySpark.
I remember @timothydijamco worked on getting both to run but I am not sure it's run by the CI now (cannot seem to find it). But perhaps this is an orthogonal issue.

The change here doesn't really do anything - we will pick up the latest pyspark version in CI with this yml file anyway (currently 3.1.1)

Reply:
Ah actually. We still testing 2.4.3 and 3.1.1 in the CI. Nothing changes with this PR.